Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -83,6 +83,7 @@ open class _QuotedChatMessageView<ExtraData: ExtraDataTypes>: _View, ThemeProvid
         contentContainerView.layer.borderWidth = 1
         contentContainerView.layer.borderColor = appearance.colorPalette.border.cgColor
         contentContainerView.layer.masksToBounds = true
+        contentContainerView.backgroundColor = appearance.colorPalette.background
```

Comment:
If I understand this correctly the rule is that the quoted messages has always the other color than the outer bubble. So this probably can’t be easily solved inside the view itself but should be set from the outside. it’s basically a reversed logic than for the main bubble color